### PR TITLE
Get config in webhook

### PR DESCRIPTION
**Pull request by rajivnathan**
Summary of changes:
1. controllers/memberoperatorconfig/configuration.go moves to toolchain-common
2. Code is updated to import config from toolchain-common
3. Updated webhook permissions and set WATCH_NAMESPACE env var
4. Updated VM mutating webhook to use configuration

Related PRs:
- https://github.com/codeready-toolchain/toolchain-common/pull/338
- https://github.com/codeready-toolchain/toolchain-e2e/pull/835

### CODE REVIEW

**Review by alexeykazakov — COMMENTED** [review]
Do we have a test with multiple comma separated keys? I don't see it...

**Comment by rajivnathan** [comment]
@alexeykazakov good catch, I meant to add it but forgot about it. Added it in fe560486139c45a4c8b9b22d08b6e17e24f84347 and made a couple fixes related to handling cases when keys are already present

File: pkg/webhook/mutatingwebhook/vm_mutate_test.go [hunk 1]
```diff
@@ -278,36 +289,49 @@ func TestEnsureVolumeConfig(t *testing.T) {
...
-
-		// when
-		userDataStr, err := addSSHKeyToUserData(userDataWithSSHKey, sshKey)
+	// given
+	for tcName, tc := range map[string]struct {
+		sshKeys                  []string
+		expectedWhenFresh        string
+		expectedWhenExistingKeys string
+	}{
+		// "single ssh key": {
```
[the remark this answers is not in the document]
Comment (rajivnathan) [reply pkg/webhook/mutatingwebhook/vm_mutate_test.go]: 🤦 fixed 5b3498cd9ccfb72ea351fe7249b2954b41309405

**Review by alexeykazakov — APPROVED** [review]

**Review by mfrancisc — APPROVED** [review]
Nice Job !

I have only few minor comments.

File: controllers/memberstatus/memberstatus_controller.go [hunk 2]
```diff
@@ -35,6 +35,8 @@ import (
 type statusComponentTag string
 
 const (
+	MemberStatusName = "toolchain-member-status"
```
[the remark this answers is not in the document]
Comment (rajivnathan) [reply controllers/memberstatus/memberstatus_controller.go:38]: Actually it's also used in https://github.com/codeready-toolchain/member-operator/blob/94559a0e01a67753246241248fdac2e4f4aec4c3/main.go#L269

File: cmd/webhook/main.go [hunk 3]
```diff
@@ -90,6 +91,13 @@ func main() {
 		setupLog.Error(err, "creating a new client failed")
 		os.Exit(1)
 	}
+
+	_, err = membercfg.GetConfiguration(cl)
+	if err != nil {
+		setupLog.Error(err, "getting member operator config failed")
+		os.Exit(1)
+	}
+
```
[the remark this answers is not in the document]
Comment (rajivnathan) [reply cmd/webhook/main.go:101]: Good idea! Done in 2f6e5f026ece2831ed749bfd56342c86587ad1e4

File: controllers/memberstatus/memberstatus_controller.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (mfrancisc) [reply controllers/memberstatus/memberstatus_controller.go:38]: Oh I see. I've missed that! Thanks!

**Pull request merged**